subsys/bluetooth/services/fast_pair/include/fp_storage.h

```diff
@@ -18,29 +20,36 @@
 extern "C" {
 #endif
 
-#include <sys/types.h>
-#include "fp_crypto.h"
+/** Length of Account Key (128 bits = 16 bytes). */
+#define FP_STORAGE_ACCOUNT_KEY_LEN	16U
+
+/** @brief Fast Pair Account Key. */
+struct fp_storage_account_key {
+	/** Account Key. */
+	uint8_t account_key[FP_STORAGE_ACCOUNT_KEY_LEN];
```

Comment:
Maybe we could also move the new definition and structure it to `fp_common.h` (or `fp_internal.h` as suggested by @kapi-no )?
So that it could be easily shared by `fp_storage` and  `fp_keys` (looking at the implementation we already know that Account Key is actually an AES key which is used to decrypt Fast Pair messages).

Reply:
We could call it simply: `struct fp_account_key` and `FP_ACCOUNT_KEY_LEN`